bedrock/firefox/views.py

```diff
@@ -620,7 +620,55 @@ def hello_screen_sharing(version):
     return version >= Version('38.0.5')
 
 
-class HelloStartView(LatestFxView):
+class HelloFxView(TemplateView):
```

Comment:
Thanks @pmclanahan - happy to defer to yourself on what you think is best for now. Closing this :)